src/main/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationService.java

```diff
@@ -993,45 +994,80 @@ void sendResponseNotificationEmailToTribunal(CaseData caseData, String caseId) {
     void sendResponseNotificationEmailToRespondent(
         CaseData caseData,
         String caseId,
-        String copyToOtherParty
+        String copyToOtherParty,
+        boolean isClaimantPseResponse,
+        String respondentIdamId
     ) {
-
-        if (DONT_SEND_COPY.equals(copyToOtherParty) || copyToOtherParty == null) {
+        // don't send email to respondents if this is a claimant PSE response and they opted out
+        if ((DONT_SEND_COPY.equals(copyToOtherParty) && isClaimantPseResponse)
+            || copyToOtherParty == null) {
             log.info("Acknowledgement email not sent to respondents");
             return;
         }
 
+
         Map<String, Object> respondentParameters = new ConcurrentHashMap<>();
         addCommonParameters(respondentParameters, caseData, caseId);
-
         respondentParameters.put(
             SEND_EMAIL_PARAMS_HEARING_DATE_KEY,
             NotificationsHelper.getNearestHearingToReferral(caseData, NOT_SET)
         );
-        respondentParameters.put(
-            SEND_EMAIL_PARAMS_EXUI_LINK_KEY,
-            notificationsProperties.getExuiCaseDetailsLink() + caseId
-        );
 
-        sendRespondentEmails(caseData, caseId, respondentParameters,
-                             notificationsProperties.getPseRespondentResponseTemplateId()
-        );
+        String emailTemplate;
+        if (isClaimantPseResponse) {
+            emailTemplate = notificationsProperties.getPseRespondentResponseTemplateId();
+            sendRespondentEmails(caseData, caseId, respondentParameters, emailTemplate);
+        } else {
+            // respondent PSE response
+            // only the current respondent gets the email
+            if (DONT_SEND_COPY.equals(copyToOtherParty)) {
+                emailTemplate = notificationsProperties.getPseClaimantResponseNoTemplateId();
+                RespondentSumTypeItem respondent = getRespondent(caseData, respondentIdamId);
+                String emailAddress = getRespondentEmail(respondent);
+                if (isBlank(emailAddress)) {
+                    log.info("Respondent does not have an email address associated with their account");
+                    return;
+                }
+                sendEmailToRespondent(emailAddress, emailTemplate, respondentParameters, caseId);
+            } else {
+                // send email to all the respondents
+                emailTemplate = notificationsProperties.getPseClaimantResponseYesTemplateId();
+                sendRespondentEmails(caseData, caseId, respondentParameters, emailTemplate);
```

Comment:
I know this is just a method you're referring to and not one you've changed but I think this may need a change. The logic that is in this doesn't take into consideration whether the parties are online or offline so there's a risk that emails could be sent to offline users